src/logging.cc

```diff
@@ -144,6 +144,8 @@ GLOG_DEFINE_string(alsologtoemail, "",
                    "in addition to logfiles");
 GLOG_DEFINE_bool(log_prefix, true,
                  "Prepend the log prefix to the start of each log line");
+GLOG_DEFINE_bool(log_year_in_prefix, true,
+                 "The year should be included in the log prefix");
```

Comment:
Isn't the following wording sufficient then?
```suggestion
                 "Include the year in the log prefix");
```

Reply:
Modified.